src/webhook/webhook.service.ts

```diff
@@ -0,0 +1,99 @@
+import { HttpService } from '@nestjs/axios';
+import {
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { CreateWebhookRequest, FetchedWebhook, WebhookEvent } from './dto';
+import { WebhookRepository } from './webhook.repository';
+import { Recipe } from '@prisma/client';
+import Cryptr from 'cryptr';
+
+@Injectable()
+export class WebhookService {
+  constructor(
+    private readonly httpService: HttpService,
+    private readonly webhookRepository: WebhookRepository,
+  ) {}
+  cryptr = new Cryptr(process.env.CRYPTR_SECRET_KEY);
+
+  async createWebhook(
+    userId: number,
+    webhookData: CreateWebhookRequest,
+  ): Promise<void> {
+    const userWebhooks = await this.webhookRepository.getAllWebhooksByUserId(
+      userId,
+    );
+    if (userWebhooks.length >= +process.env.WEBHOOK_LIMIT) {
+      throw new ForbiddenException();
```

Comment:
see [comment here](https://github.com/KubaBaniak/RecipeAppNestjs/pull/48/files#r1305294275) - you should create the forbidden response here